src/inverseRobot.js

```diff
@@ -2,12 +2,28 @@
 
 /*
  * @param {object} robot
- *
- * @return {object}
+*
+* @return {object}
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  let checkDoubleProper = false;
+
+  Object.entries(robot).forEach((key) => {
+    if (newRobot.hasOwnProperty(key[1])) {
+      checkDoubleProper = null;
```

Comment:
Instead of setting 'checkDoubleProper' to null and then returning, you should return null directly. This would stop the execution of the function and would make the code cleaner and more efficient.